virttest/bootstrap.py

```diff
@@ -262,6 +262,40 @@ def get_directory_structure(rootdir, guest_file):
         previous_indent = indent
 
 
+def sync_download_dir(interactive):
+    base_download_dir = data_dir.get_base_download_dir()
+    download_dir = data_dir.get_download_dir()
+    logging.debug("Copying downloadable assets file definitions from %s "
+                  "into %s", base_download_dir, download_dir)
+    download_file_list = glob.glob(os.path.join(base_download_dir,
+                                                "*.ini"))
+    logging.debug("Donwload file list: %s", download_file_list)
+    for src_file in download_file_list:
+        dst_file = os.path.join(download_dir,
+                                os.path.basename(src_file))
+        if not os.path.isfile(dst_file):
+            shutil.copyfile(src_file, dst_file)
+        else:
+            diff_cmd = "diff -Naur %s %s" % (dst_file, src_file)
+            diff_result = process.run(
+                diff_cmd, ignore_status=True, verbose=False)
+            if diff_result.exit_status != 0:
+                logging.info("%s result:\n %s",
+                             diff_result.command, diff_result.stdout)
+                answer = genio.ask('Download file "%s" differs from "%s". '
+                                   'Overwrite?' % (dst_file, src_file),
+                                   auto=not interactive)
+                if answer == "y":
+                    logging.debug("Restoring download file %s from sample",
+                                  dst_file)
+                    shutil.copyfile(src_file, dst_file)
+                else:
+                    logging.debug("Preserving existing %s file", dst_file)
+            else:
+                logging.debug('Download file %s exists, not touching',
```

Comment:
Yep it works fine. Please let me know what you think about the `--vt-update-config`. I think it should apply here too, but it's not strong preference. Apart from this I'm ready to merge this PR.